modules/fluentd_new/modules/fluentd_base/variables.tf

```diff
@@ -0,0 +1,227 @@
+  default     = []
+}
+
+variable "cluster_tag_key" {
+  description = "Add a tag with this key and the value var.cluster_tag_value to each Instance in the ASG."
+  type        = string
+  default     = "fluentd-servers"
+}
+
+variable "cluster_tag_value" {
+  description = "Add a tag with key var.cluster_tag_key and this value to each Instance in the ASG. This can be used to automatically find other Consul nodes and form a cluster."
+  type        = string
+  default     = "auto-join"
+}
+
+variable "termination_policies" {
+  description = "A list of policies to decide how the instances in the auto scale group should be terminated. The allowed values are OldestInstance, NewestInstance, OldestLaunchConfiguration, ClosestToNextInstanceHour, Default."
+  type        = string
+  default     = "Default"
+}
+
+variable "associate_public_ip_address" {
+  description = "If set to true, associate a public IP address with each EC2 Instance in the cluster."
+  type        = bool
+  default     = false
+}
+
+variable "tenancy" {
+  description = "The tenancy of the instance. Must be one of: default or dedicated."
+  type        = string
+  default     = "default"
+}
+
+variable "root_volume_ebs_optimized" {
+  description = "If true, the launched EC2 instance will be EBS-optimized."
+  type        = bool
+  default     = false
+}
+
+variable "root_volume_type" {
+  description = "The type of volume. Must be one of: standard, gp2, or io1."
+  type        = string
+  default     = "standard"
+}
+
+variable "root_volume_size" {
+  description = "The size, in GB, of the root EBS volume."
+  type        = number
+  default     = 50
+}
+
+variable "root_volume_delete_on_termination" {
+  description = "Whether the volume should be destroyed on instance termination."
+  default     = true
+  type        = bool
+}
+
+variable "wait_for_capacity_timeout" {
+  description = "A maximum duration that Terraform should wait for ASG instances to be healthy before timing out. Setting this to '0' causes Terraform to skip all Capacity Waiting behavior."
+  type        = string
+  default     = "10m"
+}
+
+variable "health_check_type" {
+  description = "Controls how health checking is done. Must be one of EC2 or ELB."
+  type        = string
+  default     = "EC2"
+}
+
+variable "health_check_grace_period" {
+  description = "Time, in seconds, after instance comes into service before checking health."
+  type        = number
+  default     = 300
+}
+
+variable "instance_profile_path" {
+  description = "Path in which to create the IAM instance profile."
+  type        = string
+  default     = "/"
+}
+
+variable "http_port" {
+  description = "The port to use for HTTP"
+  type        = number
+  default     = 4646
+}
+
+variable "rpc_port" {
+  description = "The port to use for RPC"
+  type        = number
+  default     = 4647
+}
+
+variable "serf_port" {
+  description = "The port to use for Serf"
+  type        = number
+  default     = 4648
+}
```

Comment:
These are Nomad related too:
https://www.nomadproject.io/docs/configuration#serf-2

Should not be required, since `fluentd` now runs directly in an EC2 instance. The corresponding security group rule should be removed too.